<launch>
    <group ns="MAV2">
	<param name="UAV_ID" value="2"/>
	<param name="safe_D" value="0.3"/>
	<param name="gamma" value="0.5"/>
	<rosparam param="laplacian">
                           [1,0,0,0,0,
                            1,1,0,0,0,
                            1,0,1,0,0,
                            1,1,0,1,0,
                            0,1,0,0,1]</rosparam>
	<param name="sub_topic" value="/vrpn_client_node/MAV2/pose"/>

        <node name="formation" pkg="flight_control" type="formation" output="screen"></node>
    </group>
</launch>
